src/algorithms/tracking/TrackPropagation.cc

```diff
@@ -305,6 +306,8 @@ namespace eicrecon {
           float pathlengthError{}; ///< Error on the pathlenght
          */
         return std::make_unique<edm4eic::TrackPoint>(edm4eic::TrackPoint{
+                                               surface,
+                                               system,
```

Comment:
And the same applies here.

Reply:
@wdconinc thank you
I will try to setup the PR now. I'll test the changes with an updated (local) EDM4eic and this change against main.